recipe/meta.yaml

```diff
@@ -123,20 +125,24 @@ outputs:
         - cuda-nvrtc
         - cuda-nvrtc-dev
         - cuda-profiler-api
+        - libcufile      # [linux]
+        - libcufile-dev  # [linux]
         - cython
         - pip
         - pyclibrary
         - python
         - setuptools
       run:
         - python
+        - cuda-pathfinder >=1.1.0,<2
         - {{ pin_compatible('cuda-version', min_pin='x', max_pin='x') }}
         # cuda-python requires cuda-nvrtc from the same major version. We ignored
         # run-exports of cuda-nvrtc-dev and instead allow a looser pinning here.
         # Same for cuda-nvcc-impl.
         - {{ pin_compatible('cuda-nvrtc', min_pin='x', max_pin='x') }}
         - {{ pin_compatible('cuda-nvcc-impl', min_pin='x', max_pin='x') }}
-        - libnvjitlink >=12.3
+        - {{ pin_compatible('libcufile', min_pin='x', max_pin='x') }}  # [linux]
+        - libnvjitlink >=12.3,<13
         - pywin32  # [win64]
```

Comment:
```suggestion
        - pywin32  # [win64]
        - numpy  # [linux]
```

Reply:
Yup. The docs mention it here: https://github.com/NVIDIA/cuda-python/blob/f60c123fb9102744c8d00deb1d2754c53d2a8c34/cuda_bindings/docs/source/module/cufile.rst?plain=1#L14

> Currently using this module requires NumPy to be present. Any recent NumPy 1.x or 2.x should work.